config/src/sumeragi.rs

```diff
@@ -16,6 +16,9 @@ const DEFAULT_ACTOR_CHANNEL_CAPACITY: u32 = 100;
 const DEFAULT_GOSSIP_PERIOD_MS: u64 = 1000;
 const DEFAULT_GOSSIP_BATCH_SIZE: u32 = 500;
 
+cfg_if::cfg_if! {
+    // not(feature = "hide_debug_config")
+    if #[cfg(debug_assertions)] {
```

Comment:
It seems overkill to pull in `cfg-if` for this. And frankly I don't think we need a complex compile-time configuration infrastructure.

Reply:
It was needed here in order to provide two versions of the view macro result.